ingest/expression_files/mtx.py

```diff
@@ -221,7 +237,9 @@ def get_mtx_dimensions(file_handler) -> List:
                     return dimensions
                 except Exception as e:
                     raise e
-        raise ValueError("MTX file did not contain data")
+        msg = "MTX file did not contain data"
```

Comment:
Right?

```suggestion
        msg = "MTX file did not contain expression data"
```

Reply:
Correct! Thanks for improving the error message to be more precise.`d18014a`